homeassistant/components/aprilaire/sensor.py

```diff
@@ -0,0 +1,263 @@
+        state_class=None,
+        status_key=Attribute.DEHUMIDIFICATION_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1],
+        value_key=Attribute.DEHUMIDIFICATION_STATUS,
+        value_fn=lambda _, value: DEHUMIDIFICATION_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="humidification_status",
+        translation_key="humidification_status",
+        device_class=None,
+        state_class=None,
+        status_key=Attribute.HUMIDIFICATION_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1, 2],
+        value_key=Attribute.HUMIDIFICATION_STATUS,
+        value_fn=lambda _, value: HUMIDIFICATION_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="ventilation_status",
+        translation_key="ventilation_status",
+        device_class=None,
+        state_class=None,
+        status_key=Attribute.VENTILATION_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1],
+        value_key=Attribute.VENTILATION_STATUS,
+        value_fn=lambda _, value: VENTILATION_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="air_cleaning_status",
+        translation_key="air_cleaning_status",
+        device_class=None,
+        state_class=None,
+        status_key=Attribute.AIR_CLEANING_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1],
+        value_key=Attribute.AIR_CLEANING_STATUS,
+        value_fn=lambda _, value: AIR_CLEANING_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="fan_status",
+        translation_key="fan_status",
+        device_class=None,
+        state_class=None,
+        status_key=None,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[],
+        value_key=Attribute.FAN_STATUS,
+        value_fn=lambda _, value: FAN_STATUS_MAP.get(value),
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Aprilaire sensor devices."""
+
+    coordinator: AprilaireCoordinator = hass.data[DOMAIN][config_entry.unique_id]
+
+    async_add_entities(
+        AprilaireSensor(coordinator, description, config_entry.unique_id)
+        for description in SENSOR_TYPES
+        if description.status_key is None
+        or description.status_sensor_exists_values is None
+        or coordinator.data.get(description.status_key)
+        in description.status_sensor_exists_values
+    )
+
+
+class AprilaireSensor(BaseAprilaireEntity, SensorEntity):
+    """Sensor entity for Aprilaire."""
+
+    entity_description: AprilaireSensorDescription
+
+    def __init__(
+        self,
+        coordinator: AprilaireCoordinator,
+        description: AprilaireSensorDescription,
+        unique_id: str | None,
```

Comment:
```suggestion
        unique_id: str,
```
You probably did this because the config entry unique id can technically be None. Please assign it to a variable and assert that variable

Reply:
Done